site/content/docs/5.3/components/side-navigation.md

```diff
@@ -0,0 +1,1425 @@
+                  <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#dashboard"/>
+                </svg>
+                Dashboard
+              </a>
+            </li>
+            <li data-bs-toggle="tooltip" data-bs-title="Charts" data-bs-placement="right">
+              <a href="#" class="side-nav-item fw-bold">
+                <svg width="1.5rem" height="1.5rem" aria-hidden="true" focusable="false">
+                  <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#chart"/>
+                </svg>
+                Charts
+              </a>
+            </li>
+            <li data-bs-toggle="tooltip" data-bs-title="Financial" data-bs-placement="right">
+              <a href="#" class="side-nav-item fw-bold">
+                <svg width="1.5rem" height="1.5rem" aria-hidden="true" focusable="false">
+                  <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#financial"/>
+                </svg>
+                Financial
+              </a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </div>
+  </div>
+  <div class="d-none d-lg-flex flex-grow-1 p-2">Content that should fill the remaining space.</div>
+</div>
+
+```html
+<button class="btn btn-primary d-lg-none" data-bs-target="#sideNavCollapsibleStatic" data-bs-toggle="offcanvas" aria-haspopup="true" aria-controls="sideNavCollapsibleStatic" id="sideNavCollapsibleStaticToggle" onclick="boosted.Collapse.getOrCreateInstance('#collapsibleElement2').show()">Show basic collapsible static side navigation</button>
+
+<div class="offcanvas-lg offcanvas-start" role="region" aria-labelledby="sideNavCollapsibleStaticToggle" id="sideNavCollapsibleStatic">
+  <div class="offcanvas-header p-0">
+    <button type="button" class="btn-close ms-auto" data-bs-target="#sideNavCollapsibleStatic" data-bs-dismiss="offcanvas" aria-expanded="true" aria-controls="sideNavCollapsibleStatic"><span class="visually-hidden">Close</span></button>
+  </div>
+  <div class="offcanvas-body h-100 p-0">
+    <nav class="side-nav side-nav-static" aria-label="Static collapsible side navigation inside an offcanvas">
+      <div class="side-nav-collapsible">
+        ...
+      </div>
+    </nav>
+  </div>
+</div>
+
+<div class="d-none d-lg-flex flex-grow-1 p-2">Content that should fill the remaining space.</div>
+```
+
+## Scrollable
+
+Even if we handle the scroll in a particular way inside our predefined themes, you may want to change this behavior with the `.side-nav-scrollable`. If you do so, please make sure that every part is accessible through scroll (the icon part and the content one).
+
+<!-- Following the spec review
+
+Here are some examples with the [collapsible with drawer content](#collapsible-with-drawer-content), that default to a scroll on both parts (the icon part and the content part) excluding the title.
```

Comment:
Note sure about why this part is inside a big comment...

Reply:
It's due to a spec review we had, removing it for sure